espnet2/enh/espnet_enh_s2t_model.py

```diff
@@ -235,7 +242,9 @@ def forward(
         perm = None
         if not bypass_enh_flag:
             ret = self.enh_model.forward_enhance(
-                speech, speech_lengths, {"num_spk": num_spk}
+                speech,
+                speech_lengths,
+                {"num_spk": num_spk, "batch_num_spk": batch_num_spk},
             )
```

Comment:
Can you add some comments here to describe the difference between "num_spk" and "batch_num_spk"?